src/System.Net.Security/src/System/Net/ContextFlagsAdapterPal.Unix.cs

```diff
@@ -23,7 +24,7 @@ public ContextFlagMapping(Interop.NetSecurityNative.GssFlags gssFlag, ContextFla
         private static readonly ContextFlagMapping[] s_contextFlagMapping = new[]
         {
             // GSS_C_INTEG_FLAG is set if either AcceptIntegrity (used by server) or InitIntegrity (used by client) is set
-            new ContextFlagMapping(Interop.NetSecurityNative.GssFlags.GSS_C_INTEG_FLAG, ContextFlagsPal.AcceptIntegrity | ContextFlagsPal.InitIntegrity),
+            new ContextFlagMapping(Interop.NetSecurityNative.GssFlags.GSS_C_INTEG_FLAG, _isServer ? ContextFlagsPal.AcceptIntegrity : ContextFlagsPal.InitIntegrity),
```

Comment:
I don't understand this.  This array initialization is happening as part of the static cctor, before any static methods are called, and thus before anyone changes the value of _isServer.